app/uk/gov/hmrc/tariffclassificationfrontend/views/appeal_choose_type.scala.html

```diff
@@ -35,6 +35,7 @@ <h2 id="appeal_choose_type-heading" class="heading-large">Choose review or a lev
    @input_radiogroup(
     field = form("type"),
     options = Seq(
+     AppealType.ADR,
```

Comment:
Linked with the ordering of enum vals in comment above. If they were in 'correct' order in the enum we could use them here with an AppealType.values.toList.ordered type call, rather than refer to them all specifically

Reply:
Probably, yes.   I just added the new one in the position required.   I guess its easier to change this way if the designers change their mind.